Patches/Dragon's Descent/ThingDefs_Races/Patch_Races_Animal_Common_Dragon.xml

```diff
@@ -0,0 +1,1121 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<!-- Making the common dragons CE compatible -->
```

Comment:
Just to make sure, you did use the CE Melee & CE Animals sheets to generate stats for the dragons, correct?

Reply:
Yes, I did use the spreadsheets. If you require the links to the copies of mine, I'll gladly link them.